lib/platform/bitbucket/index.ts

```diff
@@ -116,7 +118,17 @@ export async function getRawFile(
   // See: https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Bworkspace%7D/%7Brepo_slug%7D/src/%7Bcommit%7D/%7Bpath%7D
   const repo = repoName ?? config.repository;
   const path = fileName;
-  const url = `/2.0/repositories/${repo}/src/HEAD/${path}`;
+
+  let finalBranchOrTag = branchOrTag;
+  if (branchOrTag && branchOrTag.indexOf(pathSeparator) !== -1) {
```

Comment:
```suggestion
  if (branchOrTag?.includes(pathSeparator)) {
```
A little bit simpler  condition. 😉